models.py

```diff
@@ -26,11 +26,11 @@ class Book(db.Model):
     def serialize(self):
         return {
             'book_id': self.ourBookID,
-            'book_name_en': self.englishBookName,
-            'book_name_ar': self.arabicBookName,
-            'first_number': self.firstNumber,
-            'last_number': self.lastNumber,
-            'total_number': self.totalNumber
+            'name_en': self.englishBookName,
+            'name_ar': self.arabicBookName,
+            'first_hadith_number': self.firstNumber,
+            'last_hadith_number': self.lastNumber,
+            'total_hadith_number': self.totalNumber
```

Comment:
Let's call this one `number_of_narrations` instead. And the others `narration_start_number` and `narration_end_number`.

Reply:
@ahadith should I make change the `hadith` in other places to `narration` as well. So that. it is consistent?